scripts/db_migrator.py

```diff
@@ -832,7 +832,7 @@ def version_3_0_5(self):
         # not using hash and won't work.
         # FAST_REBOOT table exists only if fast-reboot was triggered.
         keys = self.stateDB.keys(self.stateDB.STATE_DB, "FAST_REBOOT")
-        if keys is not None:
+        if not keys:
```

Comment:
sorry. it should be `if keys:` instead of `if not keys:`

Reply:
Thanks for the catch